README.md

```diff
@@ -1,13 +1,17 @@
-# Welcome to our community...
-[![OpenEBS Welcome Banner](/images/openebs_community_banner_retro_gamer.png)](https://www.openebs.io/)
-
-## [openebs.io](https://www.openebs.io/)
+# OpenEBS Community<BR>
+> [!Important]
+> OpenEBS is an umbrella project, composed as a federation of individual sub projects (repositories). The project files in this repository apply to the umbrella project, every sub project, repository and file in the OpenEBS organization.
```

Comment:
"sub projects (repositories") needs to change to "sub projects and concerned repositories"